core/src/main/java/de/chkal/mvctoolbox/core/translation/TranslationResolver.java

```diff
@@ -0,0 +1,38 @@
+package de.chkal.mvctoolbox.core.translation;
+
+import java.util.Locale;
+
+/**
+ * An interface for resolving translations based on a key and a {@link java.util.Locale}.
```

Comment:
Maybe add a note that it is the responsibility of the implementation to lookup the current locale?

Reply:
Yes good idea, I forgot that.